payments/vipps/redirect.md

```diff
@@ -97,46 +83,257 @@ sequenceDiagram
+            "description": "Vipps Test",
+            "payeeReference": "Postman1536157124",
+            "isOperational": false,
+            "operations": []
+        }
+    }
+}
+```
+
+{:.table .table-striped}
+| Property                              | Type          | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| :------: | :------------------------------------ | :------------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|  ✔︎︎︎︎︎  | `payment`                             | `string`      | The relative URI of the payment this authorization transactions resource belongs to.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|  ✔︎︎︎︎︎  | └➔&nbsp;`authorizations.id`                           | `string`      | The relative URI of the current authorization transactions resource.                                                                                                                                                                                                                                             |
+|  ✔︎︎︎︎︎  | └➔&nbsp;`authorizations.authorizationList`                              | `array`      |  The array of authorization transaction objects.                                                                                                                                                          |
+|  ✔︎︎︎︎︎  | └➔&nbsp;`authorizations.authorizationList[]`                            | `object`      | The `authorization` transaction object described in the authorization resource below.
```

Comment:
@NullableInt has created a `transaction-response.md` include file you can use here. Ask him for help.